<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-N1W3K4HY2E"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());
      gtag('config', 'G-N1W3K4HY2E');
    </script>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta name="google-site-verification" content="BolB8QV67F58m_KZ6fCLMSyzumbg4MhwbcH_VBQU_V8"/>
    <title>phyloseminar.org | home</title>
    <link rel="canonical" href="https://www.phyloseminar.org">
    <link rel="shortcut icon" type="image/png" href="images/favicon.png">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="images/apple-touch-icon-144-precomposed.png">
    <link rel="stylesheet" type="text/css" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" type="text/css" href="phyloseminar.css">
    <link rel="alternate" type="application/rss+xml" href="index.rss" title="RSS 2.0">
    <script type="text/javascript">
      function localize_date(str, options) {
        var d = new Date(str);
        var language = window.navigator.userLanguage || window.navigator.language;
        if (typeof(options) === 'undefined') {
          options = { weekday: "long", year: "numeric", month: "long", day: "numeric", hour: "numeric", minute: "numeric", timeZoneName: "short" };
        }
        document.write(d.toLocaleString(language, options));
      }
    </script>
  </head>
  <body id="home">
    <a class="sr-only" href="#main">skip navigation</a>
<nav class="navbar navbar-default navbar-static-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button title="toggle navigation" type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">
        <img src="images/brand.png" alt="phyloseminar.org logo"/>
        <span class="sr-only">home</span>
      </a>
    </div>
    <div class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
        <li ><a style="font-size:150%;" href="index.html">home</a></li>
        
        <li ><a style="font-size:150%;" href="attending.html">attending</a></li>
        
        <li ><a style="font-size:150%;" href="recorded.html">recorded</a></li>
        
        <li ><a style="font-size:150%;" href="about.html">about</a></li>
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
        <li class="pull-left">
          <a title="YouTube" href="http://youtube.com/phyloseminar" >
            <i class="fa-brands fa-youtube icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">YouTube</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Bluesky" href="https://bsky.app/profile/phyloseminar.bsky.social" >
            <i class="fa-brands fa-bluesky icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Bluesky</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Twitter" href="http://twitter.com/phyloseminar" >
            <i class="fa-brands fa-twitter icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Twitter</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Google Groups" href="https://groups.google.com/forum/?fromgroups#!forum/phyloseminar" >
            <i class="fa-solid fa-users icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Google Groups</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="RSS" href="http://phyloseminar.org/index.rss" >
            <i class="fa-solid fa-rss icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">RSS</span>
          </a>
        </li>
        
      </ul>
    </div>
  </div>
</nav>
<main id="main" class="container" role="main">
  <h1>next on phyloseminar.org</h1>
<p class="large-body"c>
  <i>To attend a seminar, please visit <a href="https://www.youtube.com/@PhyloseminarOrg/streams">the livestream portion of our YouTube channel</a>.</i>
</p>
<h2>Next-generation sequence evolution models</h2>
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="col-md-3">
      <div>Faruck Morcos</div>
      <div><small role="presentation">UT Dallas</small></div>
    </div>
    <div class="col-md-6">
      <p>Modeling sequence evolution by learning epistatic terms from protein families</p>
    </div>
    <div class="col-md-3 text-right">
      <time datetime="2025-04-29T09:00:00-07:00">
        <script type="text/javascript">
          <!--
          localize_date("2025-04-29T09:00:00-07:00");
          //-->
        </script>
        <noscript>2025-04-29T09:00:00-07:00</noscript>
      </time>
    </div>
  </div>
  <div class="panel-body">
    <div class="col-md-3">
      <div class="portrait pull-left">
        <img class="portrait img-rounded" src="images/145morcos.jpg" width="265" height="331" alt="Faruck Morcos"/>
        <div class="text-center"><small role="presentation">Faruck Morcos</small></div>
      </div>
    </div>
    <div class="col-md-9">
      <p>The use of coevolutionary information, i.e. the knowledge of amino acid or nucleotide interactions maintained through evolution in each protein family, has provided a framework to study structural biology in a predictive manner. Amino acid couplings, obtained from inference algorithms like Direct Coupling Analysis, have been fundamental to predict non-local interactions in protein three-dimensional structure, being now the core of state-of-the-art protein structure prediction algorithms. Coevolutionary information has had an impact in understanding conformational plasticity, complex prediction in molecular interactions and specificity in signal transduction and predicting the effect of mutations. In this seminar I will discuss new directions in the study of global models of sequence evolution, primarily focused on the integration of theory and experiment. I will present recent advances in my lab integrating the statistical mechanics of sequences to develop a model of sequence evolution called Sequence Evolution with Epistatic Contributions (SEEC) that unifies several statistical features of independent models by integrating epistatic contributions and context dependance in the formulation of evolutionary landscapes. We then demonstrate how this model can produce, in silico, evolved sequences that are functional, in vivo, even after a large number of mutational events proposed by the dynamics of the model.</p>
    </div>
  </div>
</div>
<div class="panel-group" id="seminars">
  
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="row">
      <div class="col-md-3">
        <div>
          Pierre Barrat-Charlaix
        </div>
        <div><small role="presentation">Politecnico di Torino</small></div>
      </div>
      <div class="col-md-6">
        <p>Reconstruction of ancestral protein sequences using autoregressive generative models</p>
      </div>
      <div class="col-md-3 text-right">
        <time datetime="2025-05-20T09:00:00-07:00">
          <script type="text/javascript">
            <!--
            localize_date("2025-05-20T09:00:00-07:00");
            //-->
          </script>
          <noscript>2025-05-20T09:00:00-07:00</noscript>
        </time>
      </div>
    </div>
  </div>
  <div id="abstract00">
    <div class="panel-body">
      <div class="col-md-3">
        <div class="portrait pull-left">
          <img class="portrait img-rounded" src="images/146barratCharlaix.jpg" width="407" height="527" alt="Pierre Barrat-Charlaix"/>
          <div class="text-center"><small role="presentation">Pierre Barrat-Charlaix</small></div>
        </div>
      </div>
      <div class="col-md-9">
        <p>Ancestral sequence reconstruction (ASR) is an important tool to understand how protein structure and function changed over the course of evolution. It essentially relies on models of sequence evolution that can quantitatively describe changes in a sequence over time. Such models usually consider that sequence positions evolve independently from each other and neglect epistasis: the context-dependence of the effect of mutations. On the other hands, the last years have seen major developments in the field of generative protein models, which learn constraints associated with structure and function from large ensembles of evolutionarily related proteins. Here, we show that it is possible to extend a specific type of generative model to describe the evolution of sequences in time while taking epistasis into account. We apply the developed technique to the problem of Ancestral Sequence Reconstruction (ASR): given a protein family and its evolutionary tree, we try to infer the sequences of extinct ancestors.
Using both simulations and data coming from experimental evolution we show that our method outperforms state-of-the-art ones.
Moreover, it allows for sampling a greater diversity of potential ancestors, allowing for a less biased characterization of ancestral sequences.</p>
      </div>
    </div>
  </div>
</div>

  
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="row">
      <div class="col-md-3">
        <div>
          Antoine Koehl
        </div>
        <div><small role="presentation">UC Berkeley</small></div>
      </div>
      <div class="col-md-6">
        <p>Deep Models of Protein Evolution</p>
      </div>
      <div class="col-md-3 text-right">
        <time datetime="2025-05-27T09:00:00-07:00">
          <script type="text/javascript">
            <!--
            localize_date("2025-05-27T09:00:00-07:00");
            //-->
          </script>
          <noscript>2025-05-27T09:00:00-07:00</noscript>
        </time>
      </div>
    </div>
  </div>
  <div id="abstract01">
    <div class="panel-body">
      <div class="col-md-3">
        <div class="portrait pull-left">
          <img class="portrait img-rounded" src="images/147koehl.jpg" width="309" height="416" alt="Antoine Koehl"/>
          <div class="text-center"><small role="presentation">Antoine Koehl</small></div>
        </div>
      </div>
      <div class="col-md-9">
        <p>Models of protein evolution seek to quantify how proteins evolve over time while experiencing intricate constraints and adapting new functions. These models are the engine of phylogenetics, enabling, amongst other applications, phylogenetic tree reconstruction and ancestral sequence inference. Classic and contemporary work in protein sequence modeling incompletely address each others’ shortcomings - the gold standard classical models (e.g. WAG, LG) are limited by a need to consider sites in protein sequences as evolving independently, and while deep protein language models are able to account for interactions between sites, they lack an explicit time component. Here, we tackle this challenge by introducing a framework for training deep evolutionary models on protein family trees. By constructing comprehensive training datasets, we are able to train a deep generative model that bridges this methodological gap to model evolutionary transitions on unaligned sequence pairs, capturing the full spectrum of evolutionary forces including insertions and deletions. Our model, termed PEINT (Protein Evolution IN Time) significantly outperforms classical evolutionary approaches and enables realistic simulations of evolutionary trajectories. This advance opens new possibilities to understand and harness evolution for protein design, variant effect prediction, viral evolution forecasting, and statistical phylogenetics.</p>
      </div>
    </div>
  </div>
</div>

</div>

</main>

    <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.js"></script>
    <script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.js"></script>
    <script type="text/javascript" src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.js"></script>
  </body>
</html>
