<!DOCTYPE html>
<html lang="en">
  <head>
    <!-- Google tag (gtag.js) -->
    <script async src="https://www.googletagmanager.com/gtag/js?id=G-N1W3K4HY2E"></script>
    <script>
      window.dataLayer = window.dataLayer || [];
      function gtag(){dataLayer.push(arguments);}
      gtag('js', new Date());
      gtag('config', 'G-N1W3K4HY2E');
    </script>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>phyloseminar.org | home</title>
    <link rel="canonical" href="https://www.phyloseminar.org/">
    <link rel="shortcut icon" type="image/png" href="images/favicon.png">
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="images/apple-touch-icon-144-precomposed.png">
    <link rel="stylesheet" type="text/css" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.css">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.6.0/css/all.min.css" integrity="sha512-Kc323vGBEqzTmouAECnVceyQqyqdsSiqLQISBL29aUW4U/M7pSPA/gEUZQqv1cwx4OnYxTxve5UMg5GT6L4JJg==" crossorigin="anonymous" referrerpolicy="no-referrer" />
    <link rel="stylesheet" type="text/css" href="phyloseminar.css">
    <link rel="alternate" type="application/rss+xml" href="index.rss" title="RSS 2.0">
    <script type="text/javascript">
      function localize_date(str, options) {
        var d = new Date(str);
        var language = window.navigator.userLanguage || window.navigator.language;
        if (typeof(options) === 'undefined') {
          options = { weekday: "long", year: "numeric", month: "long", day: "numeric", hour: "numeric", minute: "numeric", timeZoneName: "short" };
        }
        document.write(d.toLocaleString(language, options));
      }
    </script>
  </head>
  <body id="home">
    <a class="sr-only" href="#main">skip navigation</a>
<nav class="navbar navbar-default navbar-static-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button title="toggle navigation" type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">
        <img src="images/brand.png" alt="phyloseminar.org logo"/>
        <span class="sr-only">home</span>
      </a>
    </div>
    <div class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
        <li ><a style="font-size:150%;" href="index.html">home</a></li>
        
        <li ><a style="font-size:150%;" href="attending.html">attending</a></li>
        
        <li ><a style="font-size:150%;" href="recorded.html">recorded</a></li>
        
        <li ><a style="font-size:150%;" href="about.html">about</a></li>
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        
        <li class="pull-left">
          <a title="YouTube" href="https://www.youtube.com/@PhyloseminarOrg/streams" >
            <i class="fa-brands fa-youtube icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">YouTube</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Bluesky" href="https://bsky.app/profile/phyloseminar.bsky.social" >
            <i class="fa-brands fa-bluesky icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Bluesky</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Twitter" href="http://twitter.com/phyloseminar" >
            <i class="fa-brands fa-twitter icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Twitter</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="Google Groups" href="https://groups.google.com/forum/?fromgroups#!forum/phyloseminar" >
            <i class="fa-solid fa-users icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">Google Groups</span>
          </a>
        </li>
        
        <li class="pull-left">
          <a title="RSS" href="https://www.phyloseminar.org/index.rss" >
            <i class="fa-solid fa-rss icon-2x" aria-hidden="true" role="presentation"></i>
            <span class="sr-only">RSS</span>
          </a>
        </li>
        
      </ul>
    </div>
  </div>
</nav>
<main id="main" class="container" role="main">
  <h1>next on phyloseminar.org</h1>
<p class="large-body"c>
  <i>To attend a seminar, please visit <a href="https://www.youtube.com/@PhyloseminarOrg/streams">the livestream portion of our YouTube channel</a>.</i>
</p>
<h2>Synteny</h2>
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="col-md-3">
      <div>Nadia El-Mabrouk</div>
      <div><small role="presentation">Université de Montréal</small></div>
    </div>
    <div class="col-md-6">
      <p>Reconstructing the evolution of homologous genomic regions through gain and loss events</p>
    </div>
    <div class="col-md-3 text-right">
      <time datetime="2026-03-05T09:00:00-08:00">
        <script type="text/javascript">
          <!--
          localize_date("2026-03-05T09:00:00-08:00");
          //-->
        </script>
        <noscript>2026-03-05T09:00:00-08:00</noscript>
      </time>
    </div>
  </div>
  <div class="panel-body">
    <div class="col-md-3">
      <div class="portrait pull-left">
        <img class="portrait img-rounded" src="images/154elmabrouk.jpg" width="1290" height="1428" alt="Nadia El-Mabrouk"/>
        <div class="text-center"><small role="presentation">Nadia El-Mabrouk</small></div>
      </div>
    </div>
    <div class="col-md-9">
      <p>Several biological systems rely on sets of colocalized genes sharing a common evolutionary history, and recovering such history is an important step towards elucidating many biological questions. This has been the focus of our group in the last years. We developed Synesth (for SYNteny Evolution in SegmenTal Histories) predicting the evolution of homologous genomic regions, also called syntenies, through segmental events such as duplications, horizontal transfers, gains or losses, likely involving multiple genes. The algorithm is based on a reconciliation framework comparing two topologies: a species tree and a synteny tree. Through an algebraic dynamic programming scheme, Synesth can produce various kinds of outputs (number of events, minimum cost, Pareto-optimal vectors, a single history, a set of equivalent histories, etc). However, as phylogenetic studies usually lead to gene trees, the main challenge of using Synesth is to have an appropriate synteny tree as input. Therefore, we also developed FullSynesth rather taking as input a set of gene trees and simultaneously building and reconciling a synteny supertree. Moreover, to make the problem tractable, gains were excluded from the optimization function, i.e. contrary to losses, taking advantage of simultaneous gains of multiple was disregarded. Therefore, we then studied the problem of minimizing gain and loss episodes on a tree, starting with a simple and tractable version of the Small Parsimony Problem.</p>
<p>This presentation will provide an overview of the problem, its various algorithmic aspects, computational complexity, and those related to the solution space, and we will give some examples of biological applications.</p>
    </div>
  </div>
</div>
<div class="panel-group" id="seminars">
  
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="row">
      <div class="col-md-3">
        <div>
          Darrin Schultz
        </div>
        <div><small role="presentation">Stanford University</small></div>
      </div>
      <div class="col-md-6">
        <p>Understanding animal evolution through multi-scale synteny</p>
      </div>
      <div class="col-md-3 text-right">
        <time datetime="2026-03-16T09:00:00-07:00">
          <script type="text/javascript">
            <!--
            localize_date("2026-03-16T09:00:00-07:00");
            //-->
          </script>
          <noscript>2026-03-16T09:00:00-07:00</noscript>
        </time>
      </div>
    </div>
  </div>
  <div id="abstract00">
    <div class="panel-body">
      <div class="col-md-3">
        <div class="portrait pull-left">
          <img class="portrait img-rounded" src="images/155schultz.jpg" width="421" height="525" alt="Darrin Schultz"/>
          <div class="text-center"><small role="presentation">Darrin Schultz</small></div>
        </div>
      </div>
      <div class="col-md-9">
        <p>Animal genomes record evolutionary history across multiple scales, from local gene neighborhoods to chromosome-wide architectures, but most comparative approaches still treat these layers separately. Here I present a multi-scale synteny framework that integrates microsynteny, macrosynteny, and genome-wide patterns of rearrangement to reconstruct deep animal relationships and quantify how chromosome structure evolves through time.</p>
<p>Across chromosome-scale assemblies spanning Metazoa, we model the gain and loss of syntenic blocks as a birth-death-like process to compare rates and modes of genome restructuring among lineages. We also discuss a pattern of large-scale reorganization, fusion-with-mixing (FWM), in which segments from distinct ancestral chromosomes become interleaved following fusion.</p>
<p>Characterizing chromosomal fusion-with-mixing events across animals and their close relatives revealed strong FWM signatures that supports ctenophores as sister to all other animals. While nascent, multi-scale synteny provides a unified lens to study chromosome evolution across the scale of the tree-of-life.</p>
      </div>
    </div>
  </div>
</div>

  
<div class="panel panel-default">
  <div class="panel-heading container">
    <div class="row">
      <div class="col-md-3">
        <div>
          Cedric Chauve
        </div>
        <div><small role="presentation">Simon Fraser University</small></div>
      </div>
      <div class="col-md-6">
        <p>Handling uncertainty in ancestral gene orders reconstruction</p>
      </div>
      <div class="col-md-3 text-right">
        <time datetime="2026-04-27T09:00:00-07:00">
          <script type="text/javascript">
            <!--
            localize_date("2026-04-27T09:00:00-07:00");
            //-->
          </script>
          <noscript>2026-04-27T09:00:00-07:00</noscript>
        </time>
      </div>
    </div>
  </div>
  <div id="abstract01">
    <div class="panel-body">
      <div class="col-md-3">
        <div class="portrait pull-left">
          <img class="portrait img-rounded" src="images/156chauve.jpg" width="203" height="270" alt="Cedric Chauve"/>
          <div class="text-center"><small role="presentation">Cedric Chauve</small></div>
        </div>
      </div>
      <div class="col-md-9">
        <p>Over the last thirty years, the algorithms used for reconstructing ancestral gene orders from extant gene orders have evolved from methods that did not account for gene evolution (the Small Parsimony Problem) to methods relying heavily on the availability of reconciled gene trees. In this talk I will describe how ancestral gene orders reconstruction is a problem that can be related to both genome assembly and phylogenomics. I will use this point of view to review the evolution of ancestral gene orders reconstruction methods, and how the increasing use of gene trees has drastically changed the way the problem is addressed, especially toward handling uncertainty and ambiguity in proposed ancestral gene orders.</p>
      </div>
    </div>
  </div>
</div>

</div>

</main>

    <script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.js"></script>
    <script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.js"></script>
    <script type="text/javascript" src="//netdna.bootstrapcdn.com/bootstrap/3.0.0/js/bootstrap.js"></script>
  </body>
</html>
